app/views/rentals/_form.html.erb

```diff
@@ -1,8 +1,9 @@
-<%= simple_form_for [@game, @rental]do |f| %>  <!--  not sure about the syntax here -->
+
+<%= simple_form_for [@rental] do |f| %>  <!--  not sure about the syntax here -->
```

Comment:
Not sure about this part also, but we can make like this way and change if needed later